### PR TITLE
Feature/sup 70 edit shipment modal

### PR DESCRIPTION
**Pull request by IrynaZazyba**
added opportunity to edit shipment application, dispatchers can change also source and destination location

### CODE REVIEW

File: frontend/src/components/ModalApp.js [hunk 1]
```diff
@@ -18,16 +18,20 @@ function ModalApp(props) {
...
 
+
     const onHideModal = () => {
         setErrors({
             serverErrors: '',
             validationErrors: []
         });
         setApp('');
         setCurrentItem('');
+        unavailableItems&&setUnavailableItems('');
```
Comment (eugenmolchanov) [inline frontend/src/components/ModalApp.js]: Please, format this line.

File: frontend/src/pages/application/Application.js [hunk 2]
```diff
@@ -131,13 +141,20 @@ export default () => {
             </td>
             <td><FaEdit style={{textAlign: 'center', color: '#1A7FA8'}}
                         onClick={() => {
-                            {app.type !== 'TRAFFIC' &&
```
Comment (eugenmolchanov) [inline frontend/src/pages/application/Application.js:134]: We may use simple if-else for the whole onClick function

File: frontend/src/validation/ItemPageValidationRules.js [hunk 3]
```diff
@@ -41,3 +41,9 @@ export function validateItemCategory(dto) {
 
     return errorFields;
 }
+
+
+export function checkItemsAtWarehouse(appItems, whItems) {
+    let filter = whItems.map(i => i.item.id);
```
Comment (eugenmolchanov) [inline frontend/src/validation/ItemPageValidationRules.js]: Rename filter variable.

**Review by eugenmolchanov — APPROVED** [review]
Please, see the comments.

**Review by alexeyoverchenko — APPROVED** [review]

**Pull request merged**